src/backend/src/lib.rs

```diff
@@ -436,6 +439,22 @@ fn list_custom_tokens() -> Vec<CustomToken> {
     read_state(|s| s.custom_token.get(&stored_principal).unwrap_or_default().0)
 }
 
+async fn bitcoin_ecdsa_public_key_of(canister_id: Principal, network: BitcoinNetwork) -> Vec<u8> {
```

Comment:
- Is this function suppose to be available for the outside word? Currently it's not exposed
- Why passing `canister_id` as parameter?

Reply:
One way or another, we need tests provided with this PR. So, if it's easier to provide those by adding the endpoints in this PR, let's do that.

Note: we do not need to replicate the second function you listed above - `eth_address_of` - for Bitcoin. Only the first - `caller_eth_address`.

Reply:
Ok, then I already pushed the complete PR on this (included tests)